wallet/bindings/capacitorjs/ios/WalletFramework.xcframework/Info.plist

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>AvailableLibraries</key>
+	<array>
+		<dict>
+			<key>LibraryIdentifier</key>
```

Comment:
Yes you're right, amrv7 is only for simulators (aka emulators on android)
The folder `WalletFramework.xcframework` and all his contents were created by the XCode IDE when added the xcframework with the libraries. We need an xcframework because it's the iOS way to include more than one library architecture, but by default now only install the arm64 version to be ready for production. To add armv7 we need to do it manually from XCode. In other words that code is needed only to add the armv7 lib when testing on simulators.